src/index.html

```diff
@@ -11,12 +11,106 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header-container">
+        <a
+          class="header-logo__link"
+          href="#"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+
+        <nav class="header-nav">
+          <ul class="header-nav__list">
+            <li class="header-nav__item">
+              <a
+                href="#"
+                class="header-nav__link is-active"
```

Comment:
Fix classes everywhere
```suggestion
        <nav class="nav">
          <ul class="nav__list">
            <li class="nav__item">
              <a
                href="#"
                class="nav__link is-active"
```